model/host/host.go

```diff
@@ -475,6 +475,10 @@ func (h *Host) IsEphemeral() bool {
 	return utility.StringSliceContains(evergreen.ProviderSpawnable, h.Provider)
 }
 
+func (h *Host) IsStatic() bool {
+	return h.Provider == evergreen.ProviderNameStatic
```

Comment:
Should probably check `h.Distro.Provider` instead of `h.Provider` because `h.Distro.Provider` is more widely used, and we may deprecate `h.Provider` one day (it's the same thing as `h.Distro.Provider).`